src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -9,6 +12,13 @@ namespace Raven.Server.Config.Categories
     [ConfigurationCategory(ConfigurationCategoryType.Logs)]
     public class LogsConfiguration : ConfigurationCategory
     {
+        public override void Initialize(IConfigurationRoot settings, HashSet<string> settingsNames, IConfigurationRoot serverWideSettings, HashSet<string> serverWideSettingsNames, ResourceType type,
+            string resourceName)
+        {
+            base.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
+            MicrosoftLogsPath ??= Path.Combine("MicrosoftLogs");
```

Comment:
I was wondering if we could use the same path but different filename prefix/suffix/extension? 
Don't know how would that add up to retention mechanisms though.

Thoughts?

Reply:
I would like to have it separated anyway.
But with the current implementation, we cannot put them in the same directory.
Unless we use the same LoggingSource of course.